frontends/pytorch/test/module_import/methods-debuggable-ir.py

```diff
@@ -32,8 +32,8 @@ def forward(self, x, y):
 #     the case that the name is `__main__` Torch replaces it with `__torch__` to
 #     avoid collisions.
 
-# CHECK: func private @__npcomp_priv_fn.__torch__.Submodule.forward
-# CHECK: func private @__npcomp_priv_fn.__torch__.TestModule.forward
+# CHECK-DAG: func private @__npcomp_priv_fn.__torch__.TestModule.forward
+# CHECK=DAG: func private @__npcomp_priv_fn.__torch__.Submodule.forward
```

Comment:
Typo? `CHECK-DAG`

Reply:
Didn't want to depend on the order. https://llvm.org/docs/CommandGuide/FileCheck.html#the-check-dag-directive